contextualize_pnid_files.ipynb

```diff
@@ -0,0 +1,207 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import urllib\n",
+    "from cognite.experimental import CogniteClient \n",
+    "from cognite.client.data_classes.files import FileMetadataUpdate"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "project=\"project\"\n",
+    "api_key = os.getenv(\"COGNITE_API_KEY\")\n",
```

Comment:
We don't have api keys in the environment variable here. Can you change so you  import getpass and use 
`api_key = getpass()`?